test/internet/internet.status

```diff
@@ -1,6 +1 @@
 prefix internet
-
-test-dns                          : PASS,FLAKY
-
-[$system==solaris]
```

Comment:
Maybe better to leave the section headers? Otherwise it's not so obvious how to use these status files.